sf-auth-client/src/main/java/com/ascendix/salesforce/oauth/ForceOAuthClient.java

```diff
@@ -86,8 +87,8 @@ private static void extractPartnerUrl(ForceUserInfo userInfo) {
     }
 
     private static boolean isBadTokenError(HttpResponseException e) {
-        return e.getStatusCode() == HttpStatusCodes.STATUS_CODE_SERVER_ERROR &&
-                e.getContent().contains(BAD_TOKEN_SF_ERROR_CODE);
+        return e.getStatusCode() == HttpStatusCodes.STATUS_CODE_FORBIDDEN &&
```

Comment:
I think check for `e.getStatusCode() == HttpStatusCodes.STATUS_CODE_FORBIDDEN` us just enough after API update